cluster/gce/util.sh

```diff
@@ -407,6 +407,12 @@ function detect-node-names() {
   if [[ -n "${HEAPSTER_MACHINE_TYPE:-}" ]]; then
     NODE_NAMES+=("${NODE_INSTANCE_PREFIX}-heapster")
   fi
+  # Add additional node names to the list too (if exist).
+  if [[ -n "${ADDITIONAL_MACHINE_TYPE:-}" && "${NUM_ADDITIONAL_NODES:-}" -ge 1 ]]; then
+    for ((i=1; i<=${NUM_ADDITIONAL_NODES}; i++)); do
+      NODE_NAMES+=("${NODE_INSTANCE_PREFIX}-big-$i")
```

Comment:
NIT: s/big/extra/g? 

The additional (extra) nodes don't necessary have to be big.

Reply:
done